build/Microsoft.DotNet.Cli.DependencyVersions.props

```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <CLI_MSBuild_Version>15.1.0-preview-000522-02</CLI_MSBuild_Version>
     <CLI_Roslyn_Version>2.0.0-rc3-61212-03</CLI_Roslyn_Version>
-    <CLI_NETSDK_Version>1.0.0-alpha-20170111-1</CLI_NETSDK_Version>
+    <CLI_NETSDK_Version>1.0.0-alpha-20170113-2</CLI_NETSDK_Version>
     <CLI_WEBSDK_Version>1.0.0-alpha-20170106-1-203</CLI_WEBSDK_Version>
```

Comment:
Please bump the web sdk to version 1.0.0-alpha-20170113-2-221 as well to get tests passing again